test-crawler/config.go

```diff
@@ -21,6 +21,7 @@ type Config struct {
 	Language   string     `yaml:"lang_mode"`
 	IndentJSON bool       `yaml:"indent_json"`
 	Ignore     []string   `yaml:"ignore"`
+	SingleFile string     `yaml:"single_file"`
```

Comment:
There is no use for this in the config. If you wanted to test a single file, you could simply hardcode a string in the `crawlSingleFileForMethods` method.

Instead of a single string, you have:
1. Added a line to the config files
2. Read the line from the config

Which is pointless, because the crawler will never read from the config a single file to crawl.
tl;dr remove the `single file` config line from all usages.